app/components/CardBet.tsx

```diff
@@ -104,11 +114,13 @@ export const CardBet = ({ userBets, children }: CardBetProps) => {
                 {outcomeAmountString}:
               </p>
               <div className="flex items-center space-x-1 text-sm font-semibold text-text-high-em">
-                {!market.isClosed || isWinner ? (
-                  <p>{formatValueWithFixedDecimals(outcomeBalance)}</p>
-                ) : (
-                  <p>{formatEtherWithFixedDecimals(collateralAmountSpent, 2)}</p>
-                )}
+                <p>
+                  {!market.isClosed || isWinner
+                    ? formatValueWithFixedDecimals(outcomeBalance)
+                    : market.isAnswerInvalid
+                      ? formatValueWithFixedDecimals(invalidMarketColleteralToRedeem, 2)
+                      : formatEtherWithFixedDecimals(collateralAmountSpent, 2)}
```

Comment:
I think it's happening too much here, I'm okay with letting this here for now but we should find a better place for this logic than the the **jsx**